packages/developer-console-spa/.env.example

```diff
@@ -0,0 +1 @@
+REACT_APP_API_GATEWAY=
```

Comment:
```suggestion
REACT_APP_API_GATEWAY=
PUBLIC_URL=
```

Reply:
That's cra's way of handling env variables. CRA only injects environment variables with a `REACT_APP_` prefix into the project.